frontend/src/components/SchemaEdit.vue

```diff
@@ -1,131 +1,105 @@
 <template>
-  <h2>Edit schema</h2>
-
-  <form v-if="schema">
-    <div class="mb-3">
-      <TextInput label="Name of schema" v-model="name" :args="{ id: 'name' }" />
-    </div>
-    <div class="mb-3">
-      <TextInput label="Slug" v-model="slug" :args="{ id: 'slug' }" />
-      <div id="slugHelp" class="form-text">URL-friendly ID of schema</div>
-    </div>
-
+  <div v-if="details">
+    <TextInput label="Name" v-model="details.name" :args="{ id: 'name' }"/>
+    <TextInput label="Slug" v-model="details.slug" :args="{ id: 'slug' }">
+      <template v-slot:helptext>
+        URL-friendly ID of schema
+      </template>
+    </TextInput>
     <Checkbox
-      label="Require reviews for schema and entities changes"
-      v-model="reviewable"
-      :args="{ id: 'reviewable', class: 'mb-3' }"
-    />
-  </form>
-  <h3>Attributes</h3>
-  <div v-if="schema">
-    <EditInitialAttributes
-      ref="initial"
-      :schemas="schemas"
-      :initialAttributes="clone(schema.attributes)"
-      :availableFieldNames="availableFieldNames"
-    />
-    <AddNewAttributes
-      ref="new"
-      :schemas="schemas"
-      :availableFieldNames="availableFieldNames"
-    />
+        label="Review"
+        v-model="details.reviewable"
+        :args="{ id: 'reviewable', class: 'mb-3' , tooltip: 'Require reviews for schema and entity changes?'}">
+    </Checkbox>
+    <h3 class="mt-3">Attributes</h3>
+    <EditAttributes ref="initial" :schemas="availableSchemas"
+                    :initialAttributes="clone(schema.attributes)"
+                    :availableFieldNames="availableFieldNames"/>
+    <button @click="sendData" type="button" class="mt-3 mb-3 btn btn-outline-primary">
+      <i class='eos-icons'>save</i>
+      Update schema
```

Comment:
Not technically an error but there's no visual response when updating schema so you are not sure if anything has really changed

Reply:
The idea also occurred to me. I guess I will add something :+1:

Reply:
Now, the button is disabled by default. Only after a change, the button becomes enabled.

Is this what you had in mind?

Reply:
Not exactly. This one is useful too, because it let's you know that you've actually changed something; but when you change something and click on update button you don't see any effect. It could be some popup telling that schema was updated or forced page reload to show updated schema — anything that would tell user that click on update button changed something

Reply:
Thanks for the clarification.

I was hoping to keep this PR free of new features, i.e. that I can only focus on the structure. Would you like to get everything in one PR? Or can we take care of this later in a separate PR?

Reply:
> Or can we take care of this later in a separate PR?

Sure👍

Reply:
Oops, I have added that already to this PR.... :innocent: